packages/block-editor/src/components/block-title/test/index.js

```diff
@@ -119,9 +120,9 @@ describe( 'BlockTitle', () => {
 			attributes: null,
 		} ) );
 
-		const wrapper = shallow( <BlockTitle clientId="id-name-with-label" /> );
+		render( <BlockTitle clientId="id-name-with-label" /> );
 
-		expect( wrapper.text() ).toBe( 'Reuse me!' );
+		expect( screen.getByText( 'Reuse me!' ) ).toBeVisible();
```

Comment:
We should probably check that the `'Test Label'` text can't be found in the document

Reply:
Good call, added in 822620d2875aa613afc5088d422601d296e95e9c in the 2 places where we test prioritization.